person.rb

```diff
@@ -0,0 +1,24 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name: 'Unkown', parent_permission: true)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+    @id = rand(1...10)
+  end
```

Comment:
- [ **OPTIONAL** ] Good :+1: job implementing the change required by the previous reviewer by assigning you id to random numbers. What do you thing if you increase its limit band value as it seems to be small? And if you want you can also use the `Random` **[class](https://ruby-doc.org/core-2.1.3/Random.html)** so your `@id` value can look to some thing like:
```ruby
@id = Random.rand(1...1000)
```